Tensile/cmake/TensileConfig.cmake

```diff
@@ -295,7 +295,7 @@ function(TensileCreateLibraryFiles
       # command to be invoked at build time, not cmake time.
       TensileCreateCopyTarget(
       "${Tensile_VAR_PREFIX}_LIBRARY_TARGET"
-      "${Tensile_MANIFEST_CONTENTS}"
+      "${Tensile_EMBED_LIBRARY_SOURCE};${Tensile_MANIFEST_CONTENTS}"
```

Comment:
So this is the real fix to depend on the source, right?  The earlier temp file name isn't any locking system but just adds clarity